src/Redistribution.sol

```diff
@@ -101,13 +148,23 @@ contract Redistribution is AccessControl, Pausable {
     // Maximum value of the keccack256 hash.
     bytes32 MaxH = bytes32(0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff);
 
+
+    // alpha=1% k=16 n=1M
+    uint256 public constant SAMPLE_MAX_VALUE = 94730660000000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
SAMPLE_MAX_VALUE is determined to be 1.284401 · 1072 in the storage incentives paper

Reply:
yeah, I see it has changed. thanks for the notice!